### ISSUE TITLE
Consider adding use of helmetjs by default

### ISSUE THREAD

**Issue by grtjn** [marklogic/slush-marklogic-node#287]
HelmetJS guards against several typical security attacks, and is very easy to add:

https://www.npmjs.com/package/helmet

**Comment by grtjn** [comment]
Insert this in node-app.js directly above the line with `app.use(expressSession({`:

```
var helmet = require('helmet');
app.use(helmet());
app.disable('x-powered-by');
```

**Comment by grtjn** [comment]
When using helmet like that, the hidePoweredBy is included by default, so the app.disable can be skipped in that case..

**Comment by grtjn** [comment]
For extra security, consider setting following headers (a.o. inspired by https://www.owasp.org/index.php/List_of_useful_HTTP_headers):

```
Content-Security-Policy:  default-src 'self';
X-Content-Type-Options: nosniff
X-Frame-Options: sameorigin
X-XSS-Protection: 1; mode=block
```

We should also add the HttpOnly attribute on Session cookies, which we get by default if we upgrade to express-session 1.5.0 (see also #295): https://github.com/expressjs/session#cookie-options

**Comment by grtjn** [comment]
Hmm, from the looks of it, it should be possible to cover all of these with helmetjs..

**Comment by grtjn** [comment]
This would make life a little easier though: https://github.com/helmetjs/helmet/issues/112

**Comment by EvanHahn** [comment]
I'm the maintainer of Helmet. Let me know if I can be helpful here!

**Comment by grtjn** [comment]
Thnx!

**Comment by grtjn** [comment]
The helmetjs PR is still open, pushing this back till after UI milestone..

**Comment by grtjn** [comment]
The PR has been merged, and helmetjs 2.0.0 has been released. I think it is now ready to push forward with this..
